.github/workflows/pull-request.yml

```diff
@@ -350,6 +350,14 @@ jobs:
           ~/cargo-make/makers k
           ~/cargo-make/makers logs
           ~/cargo-make/makers check-logs
+      - name: e2e test w/ Rust frontend ci-3node, source with kafka
+        timeout-minutes: 2
+        run: |
+          ~/cargo-make/makers clean-data
+          ~/cargo-make/makers ci-start ci-kafka
+          sqllogictest -p 4566 './e2e_test/v2/source/**/*.slt'
```

Comment:
Seems that this PR added `source/ddl.slt` instead of `v2/source/ddl.slt`? So the e2e case is not running at all 🤪